tasks/main.yml

```diff
@@ -5,14 +5,14 @@
 - name: test to see if selinux is running
   command: getenforce
   register: sestatus
-  changed_when: false
-  ignore_errors: true
+  failed_when: false
+  always_run: true
```

Comment:
Could you explain why do you need "always_run" function in there?
I have such tasks mostly in hardening frameworks and it's kind of annoying when they run even if I need only few tasks with specific tag.
If role is executed it will run anyway, but what is the purpose to run it "always"?